features/users/omniauth.feature

```diff
@@ -8,14 +8,29 @@ Feature: Connect to social sites
   Background:
     Given I am on the "Sign in" page
 
+  @omniauth
   Scenario: Log in with GitHub account
     When I click "with GitHub"
     Then I should see "Signed in successfully."
 
+  @omniauth
   Scenario: Log in with Google account
     When I click "with Google"
     Then I should see "Signed in successfully."
 
+  @omniauth-with-invalid-credentials
+  Scenario: Try to log in with invalid credentials
+    When I click "with Google"
+    Then I should see "invalid_credentials"
+    And I should be signed out
+
+  @omniauth-with-invalid-credentials
+  Scenario: Try to log in with invalid credentials
+    When I click "with Github"
```

Comment:
@theBoyMo I think the capitalization might need to change here, i.e. "GitHub"

Reply:
Your right, everything passes once I added the correction.

Reply:
I had one question, the error message displayed is "invalid_credentials", since that is the symbol used, and not "Invalid credentials". Is that a problem? I was assuming not since we're just testing that we have a sad path in place.